machine/machine_core/machine.py

```diff
@@ -290,8 +290,10 @@ def restart_cockpit(self) -> None:
         """Restart Cockpit.
         """
         if self.ws_container:
-            self.execute(f"podman restart {self.get_cockpit_container()}")
-            self.wait_for_cockpit_running()
+            cockpit_container = self.get_cockpit_container()
```

Comment:
TBH this doesn't make sense to me -- why/how was it working before? This smells more like an assumption that you can't call  `restart_cockpit()` before you started it once, but that shouldn't change between the two ostree versions?

Reply:
This was working before as Fedora-41 had multihost enabled and enabling it for non-default enabled machines n  `test/verify/check-shell-host-switching` calls `enable_multihost`:
```
            machine.write("/etc/cockpit/cockpit.conf",
                          '[WebService]\nAllowMultiHost=yes\n')
            machine.restart_cockpit()
```

We restart while the test never started the container:

```
        b = self.browser
        m1 = self.machines['machine1']
        m2 = self.machines['machine2']
        m3 = self.machines['machine3']

        self.enable_multihost(m1)
```